lib/sidekiq/worker_killer.rb

```diff
@@ -71,7 +71,7 @@ def call(worker, job, queue)
                                         :trunc=>false,
                                         :level=>Log4r::FATAL)
     # Note: logger.fatal won't kill the process
-    logger.fatal "Process #{::Process.pid} killed (OOM). JID: #{job['jid']}, Job: #{worker.class.name}, Args: #{job['args']}"
+    logger.fatal "Process #{::Process.pid} killed (OOM) at #{Time.now}. JID: #{job['jid']}, Job: #{worker.class.name}, Args: #{job['args']}"
```

Comment:
What's the reason not to use `Time.current`?

Reply:
Wasn't sure if there's a preference honestly. `Time.now` shows up in other places in the code, that's all I knew :)

Reply:
[TIL](https://thoughtbot.com/blog/its-about-time-zones#three-time-zones) that `Time.current` is the same as `Time.zone.now`.

Reply:
```
[1] pry(main)> Time.now
=> 2021-06-18 00:01:32 +0000
[2] pry(main)> Time.current
=> Fri, 18 Jun 2021 00:01:32 UTC +00:00
```

Reply:
Yeah, I also saw that Time.current is "easier" to work with if we have to convert timezones, so good call @keithnoguchi.